src/ad-bidders/prebid/intent-iq/intent-iq.ts

```diff
@@ -55,16 +60,27 @@ export class IntentIQ {
 				? 'fandom.com'
 				: window.location.hostname;
 
+			const intentIqConfig = {
+				partner: this.fandomId,
+				pbjs,
+				timeoutInMillis: DEFAULT_MAX_DELAY,
+				ABTestingConfigurationSource: 'percentage',
+				abPercentage: 97,
+				manualWinReportEnabled: true,
+				domainName,
```

Comment:
```suggestion
				domainName,
				...(this.intentIQVersion === newScriptVersion ? { vrBrowserBlackList: ['chrome'] } : { browserBlackList: 'Chrome' } ),
```
(nit) alternatively, kinda Reactish way

Reply:
Nice, thank you! 💎